csharp/src/Google.Protobuf/Compatibility/TypeExtensions.cs

```diff
@@ -49,6 +49,11 @@ internal static class TypeExtensions
         /// Returns true if the target type is a value type, including a nullable value type or an enum, or false
         /// if it's a reference type (class, delegate, interface - including System.ValueType and System.Enum).
         /// </summary>
+#if DOTNET35
+        internal static bool IsValueType(this Type target) {
```

Comment:
Gah, I missed the brace-at-end-of-line here. Not to worry - will fix that up later myself.